homography_decomposition.cc

```diff
@@ -0,0 +1,43 @@
+#include <iostream>
+#include <fstream>
+
+#include "colmap/base/homography_matrix.h"
+
+using namespace colmap;
+
+#include <pybind11/pybind11.h>
+#include <pybind11/stl.h>
+#include <pybind11/eigen.h>
+
+namespace py = pybind11;
+
+py::dict homography_decomposition_estimation (

```

Comment:
looks great @adisingh50. Maybe you could add a docstring here too explaining each of the inputs, and maybe explain that the points should be normalized beforehand?

Reply:
sure thing